var/spack/repos/builtin/packages/dftbplus/package.py

```diff
@@ -3,85 +3,142 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import os
-
 from spack.package import *
 
 
-class Dftbplus(MakefilePackage):
+class Dftbplus(CMakePackage, MakefilePackage):
     """DFTB+ is an implementation of the
     Density Functional based Tight Binding (DFTB) method,
     containing many extensions to the original method."""
 
     homepage = "https://www.dftbplus.org"
-    url = "https://github.com/dftbplus/dftbplus/archive/19.1.tar.gz"
-
-    version("19.1", sha256="4d07f5c6102f06999d8cfdb1d17f5b59f9f2b804697f14b3bc562e3ea094b8a8")
-
-    resource(
-        name="slakos",
-        url="https://github.com/dftbplus/testparams/archive/dftbplus-18.2.tar.gz",
-        sha256="bd191b3d240c1a81a8754a365e53a78b581fc92eb074dd5beb8b56a669a8d3d1",
-        destination="external/slakos",
-        when="@18.2:",
+    url = "https://github.com/dftbplus/dftbplus/archive/22.1.tar.gz"
```

Comment:
That's the wrong URL, there is a dedicated release artifact which is not autogenerated via git-archive.

https://github.com/dftbplus/dftbplus/releases/download/22.2/dftbplus-22.2.tar.xz